### PR TITLE
feature: email builder

### PR DESCRIPTION
**Pull request by abelhOrihuela**

### CODE REVIEW

File: admin/frontend/pages/developer-tools/email-builder/detail.js [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+          }) : []
+        },
+        content: data.content
+      })
+    } catch (error) {
+      console.error(error)
+    }
+  }
+
+  tryEmail (formData) {
```
Comment (luislizarraga) [inline admin/frontend/pages/developer-tools/email-builder/detail.js]: remove formData

File: admin/frontend/pages/developer-tools/email-builder/list.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+import Link from '~base/router/link'
+import moment from 'moment'
+
+import env from '~base/env-variables'
+import ListPageComponent from '~base/list-page-component'
+import { loggedIn } from '~base/middlewares/'
+import CreateGroup from './create'
+
+class EmailList extends ListPageComponent {
+  finishUp (data) {
```
Comment (luislizarraga) [inline admin/frontend/pages/developer-tools/email-builder/list.js]: This function isn't being used

File: admin/frontend/pages/developer-tools/email-builder/list.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import Link from '~base/router/link'
+import moment from 'moment'
+
+import env from '~base/env-variables'
+import ListPageComponent from '~base/list-page-component'
+import { loggedIn } from '~base/middlewares/'
+import CreateGroup from './create'
```
Comment (luislizarraga) [inline admin/frontend/pages/developer-tools/email-builder/list.js]: Rename to `CreateEmail`

File: lib/mailer/index.js [hunk 4]
```diff
@@ -70,13 +77,21 @@ client.sendEmail = function (conf) {
 }
 
 module.exports = class Mail {
-  constructor (template) {
+  constructor (template, isString) {
+    console.log('Class isString', isString)
```
Comment (luislizarraga) [inline lib/mailer/index.js]: Remove log

File: admin/frontend/pages/developer-tools/email-builder/list.js (hunk 3 above)
Comment (abelhOrihuela) [reply admin/frontend/pages/developer-tools/email-builder/list.js]: Done

File: lib/mailer/index.js (hunk 4 above)
Comment (abelhOrihuela) [reply lib/mailer/index.js]: Done

File: admin/frontend/pages/developer-tools/email-builder/list.js (hunk 2 above)
Comment (abelhOrihuela) [reply admin/frontend/pages/developer-tools/email-builder/list.js]: Done

File: admin/frontend/pages/developer-tools/email-builder/detail.js (hunk 1 above)
Comment (abelhOrihuela) [reply admin/frontend/pages/developer-tools/email-builder/detail.js]: Done

**Pull request merged**